advanced-vector-search/README.md

```diff
@@ -170,79 +174,48 @@ python app.py -t index -n $batch_size
+The results with the default parameters for Annoy, Faiss and NumpyIndexers are:
 
-Let's take a look at `yaml/indexer.yml`
+```bash
+python app.py -t query -i numpy
 
-```yaml
-!CompoundIndexer
-components:
-  - !NumpyIndexer
-    with:
-      index_filename: 'faiss_index.tgz'
-    metas:
-      workspace: './workspace'
-      name: wrapidx
-  - !BinaryPbIndexer
-    with:
-      index_filename: doc.gz
-    metas:
-      name: docidx
-      workspace: './workspace'
-metas:
-  name: indexer
-  workspace: './workspace'
-requests:
-  on:
-    IndexRequest:
-      - !VectorIndexDriver
-        with:
-          executor: wrapidx
-      - !KVIndexDriver
-        with:
-          executor: docidx
-    ControlRequest:
-      - !ControlReqDriver {}
+Recall@100 => 99.47000050544739%
 ```
 
-The chunk indexer is formed by a CompoundExecutor which is composed of the `FaissIndexer` and a `ChunkPbIndexer`. Having a vector indexer such as `FaissIndexer` composed of a key-value indexer is a common pattern in Jina since the vector indexer will do the similarity search and the key-value will keep track of the actual chunk values.
-
-As we can see, `FaissIndexer` receives 3 parameters:
-
-- [`index_key`]: Relates to parameters used in the faiss [index factory](https://github.com/facebookresearch/faiss/wiki/The-index-factory) which determines the types of inverted indexes and encoders used to index the vectors
-- [`index_filename`]: File name where the index is stored
-- [`train_filepath`]: Path where the train data to be indexed is stored
-
-## Evaluate the results
+```bash
+python app.py -t query -i faiss
 
-This demo also outputs the evaluation of search system. The metric is recall@k where only the true nearest neighbor is considered to be a relevant document. To that end, it computes how many times the true nearest neighbour is returned as one of the k closest vectors from a query.
+Recall@100 => 47.16999990865588%
+```
 
-With the default demo, the results are:
+```bash
+python app.py -t query -i annoy
 
-- recall@1: 0.26
-- recall@10: 0.56
-- recall@50: 0.7
-- recall@100: 0.74
+Recall@100 => 77.69999986886978%
+```
 
-Using more complex inverted indices and encoders (i.e., different `index_key`) should lead to better results. Using `index_key: 'Flat'` gives a recall equal to 1 because it is the exhaustive search mode for FAISS. To learn more about the range of supported keys and options, be sure to visit [FAISS indexes](https://github.com/facebookresearch/faiss/wiki/Faiss-indexes).
+It can be good to look for different parameters to guarantee the best results
 
 ## Wrap up
 
-In this example we have seen how to use `FaissIndexer` to use FAISS as a vector database. We also have seen how to use a pod inside a docker container inside our index and query flows.
+In this example we have seen how to ude different indexers as vector databases and how to use a `ref_indexer` as a base indexer.
```

Comment:
Little typo "use"

Reply:
can u add suggestion so that I can commit here?